samigo/samigo-app/src/webapp/jsf/author/item/imageMapQuestion.jsp

```diff
@@ -205,11 +205,11 @@
         </div>
         <h:outputLabel value="#{authorMessages.image_map_alt}" styleClass="col-md-2 form-control-label"/>
          <div class="col-md-10">
-            <sakai-text-input inputId="itemForm:imageMapAltText" 
+            <sakai-text-input name="itemForm:imageMapAltText"
```

Comment:
This should be input-id ?

Reply:
It works without setting the id. Nothing references the id as far as I can see. Did I miss something?